### ISSUE TITLE
Calcite-slider > Get screen reader to announce label formatted value on thumbnail hover

### ISSUE THREAD

**Issue by HauTruong146** [Esri/calcite-components#14847]
**monday.com sync:** #12593925737

### Check existing issues

- [x] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates and reviewed the [Accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) for guidance.

### Which Component

Slider

### Summary

Currently, when the thumbnail is in focused, the screen reader announces the value of the slider. 

With slider created with a custom label formatter for tick, the screen reader still announcing just the value.

Related issues: https://github.com/ArcGIS/business-analyst-webapp/issues/28510

### Actual Behavior

When focusing on the slider, it announces the current value of the slider, with no representation of the value might be, even with custom label formatter. 

<img width="2050" height="1217" alt="Image" src="https://github.com/user-attachments/assets/07eb21e2-c3a8-4a97-b193-60a96409cefb" />

### Expected Behavior

With the custom label formatter for individual tick value here, we can use the aria-valuetext to help screen reader announce the correct representation for the current slider value.

<img width="2066" height="1218" alt="Image" src="https://github.com/user-attachments/assets/c9c628b8-a4cf-4380-8229-430bcbd21156" />

### Reproduction Sample

https://codepen.io/HauTruong/pen/pvRZVGd

### Reproduction Steps

1. Turn on screen reader
2. Tab until the slider's thumb is in focused
3. Observe the screen reader's announcement and notice that it only announces the value

### Reproduction Version

Calcite Components 5.1.1

### Relevant Info

Chrome

MacOS VoiceOver, Windows NVDA

### Priority impact

impact - p2 - want for an upcoming milestone

### Calcite package

- [x] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

ArcGIS Business/Community Analyst

**Comment by geospatialem** [comment]
@HauTruong146 The [intended use of `slider`](https://developers.arcgis.com/calcite-design-system/components/slider/#usage) is to leverage numbers rather than text. The supplemental text for the labels can add context, but the component itself should rely on numbers to convey meaning and context.

> ### Usage
> - When offering the user a numeric adjustable value or range

Similar to the example above for the chart type, would recommend a [`segmented-control` be used, or alternatively `radio-button-group`](https://developers.arcgis.com/calcite-design-system/components/segmented-control/#component-comparison) to provide the textual context and fit of the use case.

**Comment by HauTruong146** [comment]
Gotcha, I think it's should be a fairly simple transition to segmented control or radio button group here. 

As for another use case:

<img width="386" height="1096" alt="Image" src="https://github.com/user-attachments/assets/1145a687-3789-4a45-a6d2-7ee9cbe28d6d" />

With the custom label formatter on tick here, it could be super helpful to announce the value along with some context.

**Comment by geospatialem** [comment]
> Gotcha, I think it's should be a fairly simple transition to segmented control or radio button group here.
> 
> As for another use case:
> With the custom label formatter on tick here, it could be super helpful to announce the value along with some context.

We are currently working on support for a `label` property on the component (still in development with issue: https://github.com/Esri/calcite-design-system/issues/13529 and PR: https://github.com/Esri/calcite-design-system/pull/14845). Once available the `label` could be customized to fit the use case with some additional configuration on the [`calciteSliderChange`](https://developers.arcgis.com/calcite-design-system/components/slider/#api-reference-events-calciteSliderChange) event.

**Comment by geospatialem** [comment]
@HauTruong146 Thanks for the back-and-forth discussions. Between the options above, would recommend we close out the above as unplanned. Any reservations before I close out the issue?

**Comment by HauTruong146** [comment]
Yeah, we are good on closing this issue. Thank you for your feedback.
